cypress/integration/RichTextEditor.spec.ts

```diff
@@ -302,6 +302,25 @@ describe('Rich Text Editor', () => {
         );
         expectRichTextFieldValue(expectedValue);
       });
+
+      // TODO: Cypress throwing after {enter}: Cannot resolve a Slate node from DOM node: [object HTMLSpanElement]
+      // Question: Can we fix it by upgrading the Cypress version? Is this one of those issues that happen because Cypress doesn't fire all the events Slate needs in its current version?
```

Comment:
I'd really like us to upgrade to the latest Cypress sooner than later. We are waaay behind. But that's not something trivial.

Reply:
We've tried to update it but all other tests from other packages started failing.